src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -475,13 +475,13 @@ export default class ExtraproRegistration extends Mixins(DateMixin) {
   }
 
   readonly homeIdentifierRules: Array<VuetifyRuleFunction> = [
-    (v) => !!v || 'Identifying Number is required',
-    (v) => (v && v.length <= 50) || 'Cannot exceed 50 characters'
+    (v) => !!v?.trim() || 'Identifying Number is required',
+    (v) => (v && v.trim().length <= 50) || 'Cannot exceed 50 characters'
```

Comment:
Apparently, Vuetify doesn't trim the value before applying the rules, so `trim()` is needed here to determine if the value is falsy (including empty string) and check its length.